src/Sulu/Bundle/AdminBundle/Resources/js/components/SnackbarContainer/snackbarContainer.scss

```diff
@@ -0,0 +1,13 @@
+.container {
```

Comment:
do you think we can reuse this somewhere? if this is specific to the `<Application>`, i think i would not extract a separate component

Reply:
yes inside an overlay that is way I created a component for it. While it is currented decided not to implement it, I would keep the component here.

Reply:
This is general style and the application is handling when it is used in the application context.